pkg/dogstatsd/server.go

```diff
@@ -341,6 +345,41 @@ func NewServer(demultiplexer aggregator.Demultiplexer, serverless bool) (*Server
 		}
 	}
 
+	// Creating a new logger
+	cfg := NewSeelogConfig(
+		"dogstatsd",
+		"info",
+		"common",
+		"",
+		"%n %Date %Time | [DOGSTATSD_STATSD] | [%LEV] %Msg %n",
+		false)
+
+	// Configuring the log file path
+	logFile := config.Datadog.GetString("dogstatsd_log_file")
+	if logFile == "" {
+		logFile = constants.DefaultDogstatsDLogFile
+	}
+
+	// Configuring max roll for log file, if dogstatsd_log_file_max_rolls env var is not set within datadog.yaml then default value is 3
```

Comment:
nit: two separate ways of setting this option, one is env var, the other is within `datadog.yaml`. Can simplify this by saying "if dogstatsd_log_file_max_rolls config option is not set"

Reply:
I changed the logic to:
```
if dogstatsd_log_file_max_rolls == 0 {
		config.Datadog.Set("dogstatsd_log_file_max_rolls", dogstatsd_log_file_max_rolls)
	}
```